src/about-this-guide.md

```diff
@@ -9,7 +9,8 @@ There are six parts to this guide:
 1. [Building and debugging `rustc`][p1]: Contains information that should be
    useful no matter how you are contributing, about building, debugging,
    profiling, etc.
-2. [Contributing to `rustc`][p1-5]: Contains information that should be useful
+   
+   [Contributing to `rustc`][p1-5]: Contains information that should be useful
```

Comment:
Hmm, the number of the parts should be seven if we count "Appendices" as one.

Reply:
Fixed.